rich/control.py

```diff
@@ -1,19 +1,35 @@
+import sys
 import time
 from typing import TYPE_CHECKING, Callable, Dict, Iterable, List, Union
 
+if sys.version_info >= (3, 8):
+    from typing import Final
+else:
+    from typing_extensions import Final  # pragma: no cover
+
 from .segment import ControlCode, ControlType, Segment
 
 if TYPE_CHECKING:
     from .console import Console, ConsoleOptions, RenderResult
 
-STRIP_CONTROL_CODES = [
+STRIP_CONTROL_CODES: Final = [
+    7,  # Bell
     8,  # Backspace
     11,  # Vertical tab
     12,  # Form feed
     13,  # Carriage return
 ]
-_CONTROL_TRANSLATE = {_codepoint: None for _codepoint in STRIP_CONTROL_CODES}
+_CONTROL_STRIP_TRANSLATE: Final = {
```

Comment:
We pass this dict as a default value of the `_translate_table` (hidden) argument of the `strip_control_codes()` function, but dicts are mutable so we could theoretically have weird side-effects happening - which is picked up by some code analysis tools such as Pycharm's default inspections.
Maybe we should ask MyPy/Pyright/etc. to make sure that we never accidentally mutate these values, with such annotations?